docs/python-reference/prairielearn/core.md

```diff
@@ -0,0 +1,7 @@
+<!-- prettier-ignore -->
+::: prairielearn.core
+    options:
+        members:
+            - from_json
+            - to_json
+            - _JSONPythonType
```

Comment:
Should we really document this type?

Reply:
I included it for completeness with `from_json` and `to_json`. I can remove it though.

Reply:
Going to remove it, probably doesn't add much value.

Reply:
Removed in ce5a852